py/torch_tensorrt/dynamo/conversion/impl/pool.py

```diff
@@ -0,0 +1,106 @@
+from typing import Optional, Sequence, Union
+
+# @manual=//deeplearning/trt/python:py_tensorrt
+import tensorrt as trt
+from torch.fx.node import Target
+from torch_tensorrt.dynamo.conversion.converter_utils import extend_attr_to_tuple
+from torch_tensorrt.fx.converters.converter_utils import (
+    SourceIR,
+    has_dynamic_shape,
+    set_layer_name,
+)
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+
+
+def avg_poolNd(
+    network: TRTNetwork,
+    target: Union[Target, str],
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    kernel_size: Sequence[int],
+    stride: Union[int, Sequence[int]],
+    padding: Union[int, Sequence[int]] = 0,
+    ceil_mode: bool = False,
+    count_include_pad: bool = True,
+    divisor_override: Optional[int] = None,
+) -> TRTTensor:
+    if has_dynamic_shape(input.shape):
+        assert input.shape[1] != -1, "Channel dim can't be dynamic for pooling."
+
+    if ceil_mode is not False:
+        raise RuntimeError("ceil_mode is not yet supported!")
+
+    if divisor_override is not None:
+        raise RuntimeError("divisor_override is not yet supported!")
+
+    dim = len(kernel_size)
+
+    kernel_size = extend_attr_to_tuple(kernel_size, dim)
+
+    if stride is None:
+        stride = kernel_size
+    else:
```

Comment:
By the function documentation and the [schema](https://github.com/pytorch/pytorch/blob/6aae636f69abebfb8cffea9f7694f9d87929faf9/aten/src/ATen/native/native_functions.yaml#L3736), it should not be allowed for stride to be `None`, so this can be removed

Reply:
Thanks for pointing it out! According to the schema, the stride could be `[]` by default, so I made changes accordingly.